.github/workflows/deploy-services-sandbox.yml

```diff
@@ -0,0 +1,81 @@
+name: Deploy Service's Test Sandbox Stack to AWS
+
+on:
+  push:
+    branches:
+      - services-hyp3-sandbox
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - environment: hyp3-services-test
+            domain: hyp3-services-test.asf.alaska.edu
+            template_bucket: services-hyp3-artifacts
+            image_tag: test
+            product_lifetime_in_days: 14
+            default_credits_per_user: 0
+            default_application_status: APPROVED
+            cost_profile: DEFAULT
+            deploy_ref: refs/heads/develop
```

Comment:
The `deploy_ref` field is actually only required in [deploy-daac.yml](https://github.com/ASFHyP3/hyp3/blob/develop/.github/workflows/deploy-daac.yml) to allow deploying different environments from different branches, I plan to remove this line from our other deploy workflows where it's not needed.
```suggestion
```